### PR TITLE
update to buildkit 0.13

### PR DESCRIPTION
**Pull request by steved**

### CODE REVIEW

File: pkg/buildkit/buildkit.go [hunk 1]
```diff
@@ -374,26 +385,21 @@ func (c *Client) ResolveAuth(registryHostname string) (authn.Authenticator, erro
...
-	})
+	defer func() {
+		<-pw.Done()
+	}()
 
 	var imageName string
 
 	eg.Go(func() error {
-		res, err := c.bk.Solve(ctx, nil, so, ch)
+		res, err := c.bk.Solve(ctx, nil, so, pw.Status())
```
[the remark this answers is not in the document]
Comment (steved) [reply pkg/buildkit/buildkit.go:402]: You're proposing it, so I would guess you think so, but why do you think it's better? I like the idea of us not managing statusChan and having a clear `<-pw.Done()` interface.

Or is just a minimal difference from what we currently have.

[the remark this answers is not in the document]
Comment (fraenkel) [reply pkg/buildkit/buildkit.go:402]: it actually follows what buildkit did and now does.
https://github.com/moby/buildkit/blob/b6bd514373b1db2812235841d013ef67bfd092a9/examples/build-using-dockerfile/main.go#L104

this was the previous code
https://github.com/moby/buildkit/blob/37131781d719f0ed5a182ebf6b2f6e91288bcd28/examples/build-using-dockerfile/main.go#L103

[the remark this answers is not in the document]
Comment (steved) [reply pkg/buildkit/buildkit.go]: Oh, interesting. I guess we also get better control over the waitgroup with `UpdateFrom`. updated!

[the remark this answers is not in the document]
Comment (fraenkel) [reply pkg/buildkit/buildkit.go]: that was another reason.

**Pull request merged**